org.lflang/src/org/lflang/cli/CliBase.java

```diff
@@ -36,22 +50,42 @@
  * @author Atharva Patil
  */
 public abstract class CliBase implements Runnable {
+    /**
+     * Models a command specification, including the options, positional
+     * parameters and subcommands supported by the command.
+     */
+    @Spec CommandSpec spec;
 
     /**
      * Options and parameters present in both Lfc and Lff.
      */
-    @Parameters(
-        arity = "1..",
-        paramLabel = "FILES",
-        description = "Paths of the files to run Lingua Franca programs on.")
-    protected List<Path> files;
+    static class MutuallyExclusive {
+         @Parameters(
+         arity = "1..",
+         paramLabel = "FILES",
+         description = "Paths of the files to run Lingua Franca programs on.")
```

Comment:
I am confused by this. Did you mean something like this?
```suggestion
         description = "Paths to one or more Lingua Franca programs.")
```